setup.py

```diff
@@ -14,13 +14,9 @@
 
 install_requires = [
     "django>=3,<4",
-    # Todo: determine the versioning strategy PyFCM is using and adjust this if necessary.
-    # The last backward incompatible change seems to have been made in a minor version
-    # (1.4.0) so for now we'll assume we're safe within a minor version - and both 1.4.x
-    # and 1.5.x currently look good.
-    # https://github.com/olucurious/PyFCM/issues/285
-    "pyfcm>=1.4,<1.6",
+    "pyfcm==2.0.6",
```

Comment:
Can we loosen the pyfcm dependency?

Reply:
Perhaps. I'm a bit wary of the author's versioning. He isn't very active and there doesn't seem to be much oversight on PRs. I haven't looked too deeply at the test suites for this repo, and I've never released a library before so I'm not too certain of best practices here, but I think we'd want to ensure the method signatures and handling of mocked responses are consistent across a wide range of versions.

I have another PR in my head that I'd like to submit to PyFCM around [optional arguments](https://github.com/olucurious/PyFCM/issues/357) - maybe we can circle back to this in a couple weeks when I've got that merged in and there's a new PyFCM version released?

Reply:
Update: My second PR was merged and released in 2.0.7. However he hasn't authored any release notes since 2.0.1 when the breaking google API changes were addressed. I think we're betting off erring on the side of caution and pinning `2.0.7`.